packages/rest/src/providers/send.provider.ts

```diff
@@ -16,9 +15,10 @@ import {writeResultToResponse} from '../writer';
  * @returns The handler function that will populate the
  * response with operation results.
  */
+
 @injectable({scope: BindingScope.SINGLETON})
-export class SendProvider implements Provider<Send> {
-  value() {
+export class SendProvider {
+  static value() {
```

Comment:
not necessarily in this PR, can we also turn `SendResponseMiddlewareProvider` into a static Provider?

Reply:
It's not needed as `SendResponseMiddlewareProvider` is a singleton.